Vungle/VungleAdapter/ALVungleMediationAdapter.m

```diff
@@ -893,80 +921,73 @@ - (instancetype)initWithParentAdapter:(ALVungleMediationAdapter *)parentAdapter
     return self;
 }
 
-- (void)bannerAdDidLoad:(VungleBanner *)banner
+- (void)bannerAdDidLoad:(VungleBannerView *)bannerView
 {
-    [self.parentAdapter log: @"AdView loaded: %@", banner.placementId];
-    
-    NSString *creativeIdentifier = banner.creativeId;
+    [self.parentAdapter log: @"AdView loaded: %@", bannerView.placementId];
+
+    NSMutableDictionary *extraInfo = [NSMutableDictionary dictionaryWithCapacity: 3];
+
+    NSString *creativeIdentifier = bannerView.creativeId;
     if ( [creativeIdentifier al_isValidString] )
     {
-        [self.delegate didLoadAdForAdView: self.parentAdapter.adViewContainer withExtraInfo: @{@"creative_id" : creativeIdentifier}];
+        extraInfo[@"creative_id"] = creativeIdentifier;
     }
-    else
+
+    // TODO: We are confirming with MAX if we need to pass bannerView's w and h through this callback or not.
+    CGSize adSize = [bannerView getBannerSize];
+    if ( !CGSizeEqualToSize(CGSizeZero, adSize) )
     {
-        [self.delegate didLoadAdForAdView: self.parentAdapter.adViewContainer];
```

Comment:
u still need this if the extraInfo is empty.

Reply:
Updated.

I copied AdMob code here as: https://github.com/AppLovin/AppLovin-MAX-SDK-iOS/blob/007c74c0e154e5b983f846b31226c2e663885b68/Google/GoogleAdapter/ALGoogleAdViewDelegate.m#L37-L61

Do you think I need to check AL SDK version?
```if ( ALSdk.versionCode >= 6150000 )

Reply:
Updated.